src/ert/gui/tools/file/file_dialog.py

```diff
@@ -58,6 +60,7 @@ def __init__(
 
     @Slot()
     def _stop_thread(self):
+        self._file.close()
```

Comment:
Do you think that `_stop_thread` could be renamed after the addition of `_file` there, e.g. `_stop_thread_and_close_file`? Or stopping the thread implies already in closing the file 🤔

Reply:
Hmm, you think that using the term `stop` implies that we might restart it (and if so the file should perhaps not be closed). Maybe we can change to `_quit_thread()` as `quit()` is the thread call we actually run. That implies that files should be closed (and they might be so under the hood anyway in the quit code).

Reply:
Nice solution!

Reply:
Good catch, will check!

Reply:
It failed locally, so fixed. Did not understand why it didn't fail in CI.